plotter/bubbles_test.go

```diff
@@ -76,19 +63,40 @@ func ExampleBubbles() {
 	p.X.Label.Text = "X"
 	p.Y.Label.Text = "Y"
 
-	bs, err := NewBubbles(bubbleData, vg.Points(1), vg.Points(20))
+	sc, err := NewScatter(scatterData)
 	if err != nil {
 		log.Panic(err)
 	}
-	bs.Color = color.RGBA{R: 196, B: 128, A: 255}
-	p.Add(bs)
 
-	err = p.Save(200, 200, "testdata/bubbles.png")
+	// Specify style for individual points.
+	sc.GlyphStyleFunc = func(i int) draw.GlyphStyle {
+		c := color.RGBA{R: 196, B: 128, A: 255}
+		var minRadius, maxRadius = vg.Points(1), vg.Points(20)
+		rng := maxRadius - minRadius
+		_, _, z := scatterData.XYZ(i)
+		d := (z-minZ) / (maxZ - minZ)
+		r := vg.Length(d)*rng + minRadius
+		return draw.GlyphStyle{Color: c, Radius: r, Shape: draw.CircleGlyph{}}
+	}
+	p.Add(sc)
+
+	img := vgimg.New(200, 200)
```

Comment:
This doesn't need to be changed from `err = p.Save(200, 200, "testdata/bubbles.png")` as far as I can tell